includes/class-wp-agora-io-channel.php

```diff
@@ -239,11 +284,17 @@ public function save( $args ) {
       $recordingSettings[$key] = sanitize_text_field($args[$key]);
       return $recordingSettings[$key];
     }, array_keys(self::$defaultRecordingSettings));
-
+    
     update_post_meta($post_id, 'channel_video_settings', $videoSettings);
```

Comment:
Why aren't these inputs sanitized? all the new inputs are sanitized, why not update the old ones?